medmodels/medrecord/_overview.py

```diff
@@ -161,7 +163,10 @@ def _extract_string_attribute_info(
     if (len(values) > max_number_values) | (len(values_string) > max_line_length):
         values_string = f"{len(values)} {long_string_suffix}"
 
-    return {"values": values_string}
+    return {
+        "type": "Categorical" if short_string_prefix == "Categories" else "Text",
```

Comment:
not a big fan of mixing types available in `AttributeType`s with custom types like `Text`. Imo, it should just use the types in `AttributeType`. Otherwise it gets confusing